test/src/main/scala/scalarules/test/resources/strip/BUILD

```diff
@@ -14,3 +15,12 @@ scala_specs2_junit_test(
     unused_dependency_checker_mode = "off",
     deps = [":noSrcsWithResources"],
 )
+
+scala_specs2_junit_test(
+    name = "resouceStripPrefixFromExternalRepoTest",
+    size = "small",
+    srcs = ["ResourceStripPrefixTest.scala"],
+    suffixes = ["Test"],
+    unused_dependency_checker_mode = "off",
+    deps = ["@io_bazel_rules_scala//test/src/main/scala/scalarules/test/resources/strip:noSrcsWithResources"],
```

Comment:
This feels a bit brittle to me. In Bazel `io_bazel_rules_scala` should be equivalent to this repo (see `experimental_remap_main_repo` flag in Bazel) and I think it's just a matter of time until it's on by default. By then this test won't mean anything.
Maybe add another `local_repository` in our WORKSPACE just for testing and add there one target with the strip resource prefix? Duplication but worth it for confidence.

Reply:
@ittaiz you mean something like this https://github.com/bazelbuild/rules_scala/pull/944/commits/edafc20c260c43cebfdd8194ce4bb47117b80de0
I'm really not sure I did good job integrating test into the build